### PR TITLE
Update Charter / Readme.md

### PR DESCRIPTION
**Pull request by darcyclarke**
- remove reference to legacy nsp (Node Security Project) which was acquired by npm inc. (which was then acquired by GitHub)
- remove "ecosystem" prefix for the WG as this has been promoted & is canonically known as Node.js's "Security WG"

Notably, this does remove an item from the "Charter" but that may be reasonable/within scope.

/cc @mhdawson

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mhdawson** [comment]
Went to look at governance etc. related to changing. I think we should change here first:

https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md#ecosystem-security

@darcyclarke interested/have time to submit PR there or would you like me to do it?

**Review by fraxken — APPROVED** [review]
LGTM

**Review by sheelpriy — APPROVED** [review]
LGTM

**Review by hebertcisco — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 [![Security WG Twitter Hashtag](https://img.shields.io/badge/Twitter-%23SecurityWG-blue.svg)](https://twitter.com/search?q=SecurityWG)
 [![OpenJS Slack Invite](https://img.shields.io/badge/join%20slack%20on-nodejs--security--wg-green.svg)](https://slack-invite.openjsf.org/)
 [![OpenSSF scorecard](https://api.securityscorecards.dev/projects/github.com/nodejs/security-wg/badge)](https://api.securityscorecards.dev/projects/github.com/nodejs/security-wg)
-# Ecosystem Security Working Group
+
+# Security Working Group
```
Comment (mhdawson) [inline README.md:7]: This and similar places should now be updated to say `team` instead of `working group`.

**Pull request merged**